<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpaceFab - Software Controller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/product-SMALL-geo-pmac.jpg</normaloff>:/product-SMALL-geo-pmac.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="ServerOnOffButton">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>171</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Switch on server</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ServerLabel">
        <property name="text">
         <string>    Server Off</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="ClickableLabel" name="StatusQuestion">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resource.qrc">:/Information.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="ConnectToGeobrickButton_1">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>171</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Connect to GeoBrick 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QLabel" name="GeobrickConnectedLabel">
        <property name="text">
         <string>    Disconnected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="ConnectToGeobrickButton_2">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>171</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Connect to GeoBrick 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="DisconnectToGeobrick">
        <property name="text">
         <string>Disconnect Geobrick</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabUserInterface">
       <attribute name="title">
        <string>User interface - SpaceFab</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabSpaceFabExpert">
       <attribute name="title">
        <string>Expert mode - SpaceFab</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="7" column="2">
           <widget class="QLabel" name="variable_label_8">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q94 = undef</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="variable_label_9">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q95 = undef</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="variable_label_3">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q78 = undef</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QLabel" name="variable_label_11">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    p91 = undef</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="variable_label_2">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q77 = undef</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="variable_label_7">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q89 = undef</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QPushButton" name="SendCommandButton">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="12" colspan="2">
           <widget class="QTextEdit" name="textEditExpertMode"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="variable_label_4">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q79 = undef</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="variable_label_6">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q88 = undef</string>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QLabel" name="variable_label_12">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    p80 = undef</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="variable_label_5">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q87 = undef</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="variable_label_1">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q70 = undef</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLineEdit" name="lineEditExpertMode"/>
          </item>
          <item row="9" column="2">
           <widget class="QLabel" name="variable_label_10">
            <property name="maximumSize">
             <size>
              <width>159</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>    q96 = undef</string>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QPushButton" name="RetrieveCommonVariables">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
